src/main/java/org/monarchinitiative/biodownload/ftp/NIOftp.java

```diff
@@ -66,13 +66,11 @@ public static File ftp(final URL from, File target) throws FileDownloadException
         return downloadedFile;
     }
 
-    private static FileReader createReader(final URL from) throws FileDownloadException {
-        assert !Objects.isNull(from);
+    private static FileReader createReader(URL from) throws FileDownloadException {
         return new FileReader(from);
     }
 
     private static FileWriter createWriter(File target) throws FileDownloadException {
-        assert target != null;
```

Comment:
Don't we want to make sure that target is not null?

Reply:
It's never `null` as the method is private and it is null-checked before

Reply:
Moreover, assertions are disabled by default. So we should either do explicit null check with something like `Objects.requireNonNull`, or just leave it be. I think it should be OK to drop this assertion.